docs/04/00.md

```diff
@@ -30,6 +30,8 @@ The template plugin will prompt each property that needed to complete the scaffo
 className [Application]:
 ```
 
+The `g8Scaffold` command will not overwrite existing files.
```

Comment:
Someone wants to translate that to [Japanese](https://github.com/foundweekends/giter8/blob/develop/docs/ja/04/00.md) and/or [Korean](https://github.com/foundweekends/giter8/blob/develop/docs/ko/04/00.md)?